contracts/token/psp1155/psp1155.rs

```diff
@@ -234,6 +249,8 @@ impl<T: PSP1155Storage + Flush> PSP1155Internal for T {
             return Ok(())
         }
 
+        self._before_token_transfer(Some(&from), None, &ids_amounts)?;
```

Comment:
You can remove the logic above related to allowance. It is permissioned function so it is okay if it will ignore that.

Reply:
Are you talking about this?
```rs
if from != operator && !self._is_approved_for_all(from, operator) {
    return Err(PSP1155Error::NotAllowed)
}
```

Reply:
Yes=)